lua/tests/infoview_lsp_spec.lua

```diff
@@ -0,0 +1,62 @@
+local infoview = require('lean.infoview')
+local helpers = require('tests.helpers')
+
+local function get_info_lines(infoview_info)
+  return table.concat(vim.api.nvim_buf_get_lines(infoview_info.bufnr, 0, -1, true), "\n")
+end
+
+local function infoview_lsp_test(pos, expected)
+    local infoview_info = infoview.open()
+    vim.api.nvim_win_set_cursor(0, pos)
+    infoview.update(infoview_info.bufnr)
```

Comment:
Also tried replacing this update with `vim.wait(5000)`, but couldn't get `CursorHold` to trigger for some reason - though I do prefer it this way because it's more like a unit test. Probably best to test `CursorHold` separately (`TODO`).

Reply:
Yeah I couldn't get CursorHold to fire either when I tried this awhile ago... which is why I instead wrote an explicit test helper to crank (i.e. update) the infoview, and didn't depend on it happening on hold.